README.md

```diff
@@ -34,6 +34,12 @@ swiftlint.lint_files
 If you want the lint result shows in diff instead of comment, you can use `inline_mode` option. Violations that out of the diff will show in danger's fail or warn section.
 
 ```rb
+swiftlint.directory "Directory A"
+```
+
+Lint different directories individually
```

Comment:
I would make this a full sentence to match the rest of the readme – something like:

> The `directory` option does ...

What would've been the most helpful when you were reading it?

Reply:
You are absolutely right, this should be a full sentence. I struggled writing this as I wasn't convinced the directory option works as expected.

My understanding from issue #41 was and looking at danger_plugin.rb that directory was meant to check the whole contents of a directory provided, but at the same time this would never happen as https://github.com/ashfurrow/danger-swiftlint/blob/master/lib/danger_plugin.rb#L127 shows that files is always `nil` and therefore the directory parameter is being ignored. 

That was at least my understanding of the source code, but I'm not a ruby dev and therefore wasn't 100% confident my understanding is right. 

If my understanding of that is right, I'm happy to remove that part here as I won't be factual correct.

Reply:
Hmm, good question. To be honest I haven't touched the code in a while and had to look it up.

The `directory` option is meant to specify the current working directory (`pwd`) that `swiftlint` gets executed in. This lets folks have `.swiftlint.yml` files in subdirectories that are specific to, say, tests. It doesn't lint the entire contents of the directory, just the changed files in there.

I think this is correct, but could @thiagofelix take a look and make sure?